generator/generate.go

```diff
@@ -269,6 +271,24 @@ type SearchIndex struct {
 	Specs map[string][]string `json:"Specs"`
 }
 
+func minifyHTML(htmlFileName string) {
+	if !env.ShouldMinifyReports() {
+		return
+	}
```

Comment:
Can you make this check outside and before invoking this method?

Reply:
`minifyHTML` is used in two places that's the reason I have put that check inside this function it self. May be we can rename it to `minifyHTMLIfNeeded` or ``minifyHTMLIfEnabled`